proj.py

```diff
@@ -0,0 +1,123 @@
+    plt.show()
+
+
+def year_added():
+    """sorts and displays the data acc. to the year they were added to Netflix."""
+    DF['year_added'] = DF['date_added'].dt.year
+    year_sort = DF.groupby('year_added')['type'].value_counts(normalize=True) * 100
+    print(year_sort)
+
+
+def release_sort():
+    """sorts and displays the data acc. to the year they were released to the public."""
+    cf = DF.groupby(['release_year', 'type'])['type'].value_counts().sort_values(ascending=False)
+    print(cf)
+
+
+def country_sort():
+    """sorts and displays the data acc. to the country they were released in."""
+    cf = DF.groupby(['country', 'type'])['type'].value_counts().sort_values(ascending=False)
+    print(cf)
+
+
+def listed_sort():
+    """sorts and displays the data acc. to the category they were listed in."""
+    cf = DF.groupby(['listed_in'])['type'].value_counts()
+    print(cf)
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-r', '--ratings', action='store', dest='ratings', help='Shows plot for ratings.', nargs='?',
+                        const=1)
+    parser.add_argument('-t', '--typeComparison', action='store', dest='frequency_tr', help='Shows comparison between '
+                                                                                            'type and ratings.',
+                        nargs='?', const=2)
+    parser.add_argument('-y', '--yearAdded', action='store', dest='year_added', help='Shows frequency of Movies and '
+                                                                                     'TV Shows released '
+                                                                                     'in years.', nargs='?', const=3)
+    parser.add_argument('-s', '--releaseSort', action='store', dest='release_sort', help='Shows count of Movies and TV '
+                                                                                         'Shows sorted acc. '
+                                                                                         'to release year.', nargs='?',
+                        const=3)
+    parser.add_argument('-c', '--countrySort', action='store', dest='country_sort', help='Shows count of Movies and '
+                                                                                         'TV Shows '
+                                                                                         'sorted acc. to countries.',
+                        nargs='?', const=4)
+    parser.add_argument('-l', '--listedSort', action='store', dest='listed_sort', help='Shows count of Movies '
+                                                                                       'and TV Shows sorted acc. '
+                                                                                       'to the category of listing.',
+                        nargs='?', const=5)
+
+    DF, con, c = init_app()
```

Comment:
How is `DF` working? it's a local variable and you are accessing it as a global variable. Convert this to lowercase and pass it to functions specifically where it is needed.

Reply:
DF converted to lowercase and passed to the functions where required.